sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -630,8 +630,8 @@ _pi_event::_pi_event(pi_context context, CUevent eventNative)
     : commandType_{PI_COMMAND_TYPE_USER}, refCount_{1}, has_ownership_{false},
       hasBeenWaitedOn_{false}, isRecorded_{false}, isStarted_{false},
       streamToken_{std::numeric_limits<pi_uint32>::max()}, evEnd_{eventNative},
-      evStart_{nullptr}, evQueued_{nullptr}, queue_{nullptr},
-      context_{context} {
+      evStart_{nullptr}, evQueued_{nullptr}, queue_{nullptr}, context_{
```

Comment:
Would avoid changing code I don't think we've touched (assuming this was some clang-format generated change)